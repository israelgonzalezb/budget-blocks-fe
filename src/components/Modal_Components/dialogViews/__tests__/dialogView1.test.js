
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import dialogView1 from '../dialogView1';


describe('dialogView1 test', () => {
  it('dialogView1 should match snapshot', () => {
    const component = renderer.create(<dialogView1
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
