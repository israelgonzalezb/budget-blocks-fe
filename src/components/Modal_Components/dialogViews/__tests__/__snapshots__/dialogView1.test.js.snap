// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dialogView1 test dialogView1 should match snapshot 1`] = `<dialogView1 />`;
